src/otx/algo/segmentation/backbones/litehrnet.py

```diff
@@ -1435,14 +1247,7 @@ class LiteHRNetBackbone:
 
     LITEHRNET_CFG: ClassVar[dict[str, Any]] = {
         "lite_hrnet_s": {
-            "stem": {
-                "stem_channels": 32,
-                "out_channels": 32,
-                "expand_ratio": 1,
-                "strides": [2, 2],
-                "extra_stride": True,
-                "input_norm": False,
-            },
+            "stem": Stem(extra_stride=True),
```

Comment:
I'm a little bit concerned that setting `Stem` as class variable. Because it's shared to all class instances, if one of instances changes value in the `Stem`, other insance which uses `Stem` should be affected. How about leaving it as it is for now and discuss about it to find better design?

Reply:
I will move it to __new__ method.

Reply:
moved https://github.com/openvinotoolkit/training_extensions/pull/3863/commits/e7ea93c460a67587eaab5362b65686150531053a